src/sections/Home/Banner-4/index.js

```diff
@@ -7,45 +7,70 @@ import Button from "../../../reusecore/Button";
 import VintageBox from "../../../reusecore/VintageBox";
 
 import { FaMapMarkedAlt } from "@react-icons/all-files/fa/FaMapMarkedAlt";
-import { FiDownloadCloud } from "@react-icons/all-files/fi/FiDownloadCloud";
 
 import Banner1SectionWrapper from "./banner4.style";
 
 import meshmapVideo from "../../../assets/video/meshery/meshmap/meshmap-short-consul.mp4";
+import { graphql, useStaticQuery } from "gatsby";
+import { getImage } from "gatsby-plugin-image";
+import { BgImage } from "gbimage-bridge";
 
 const Banner1 = () => {
+
+  const { backgroundImage123 } = useStaticQuery(
+    graphql`
+      query {
+        backgroundImage123: file(
+          relativePath: { eq: "poster.png" }
+        ) {
+          childImageSharp {
+            gatsbyImageData(
+              width: 2000
+              quality: 50
+              webpOptions: { quality: 70 }
```

Comment:
Whoa. Fancy. 

Teach me, oh, great and wise, MeshMate Nikhil.

Reply:
Haha, sure.
Every week on Monday at 5:30 PM IST. 😂